lib/internal/bootstrap_node.js

```diff
@@ -304,6 +306,36 @@
     };
   }
 
+  function setupProcessICUVersions() {
+    const icu = process.binding('config').hasIntl ?
+      process.binding('icu') : undefined;
+    if (!icu) return;  // no Intl/ICU: nothing to add here.
+    // With no argument, getVersion() returns a comma separated list
+    // of possible types.
+    const versionTypes = icu.getVersion().split(',');
```

Comment:
Why not have `icu.getVersion()` simply return an Array? Or perhaps (even better) an object with the keys and values already set so that there is only one call to the binding layer?